### PR TITLE
NavTop component Styles

### PR DESCRIPTION
**Pull request by mariokleen**
I've tried

**Preview**

image

image

image

image

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.scss [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+
+.menu-item.MuiMenuItem-root {
+    font-size: var(--tx-S);
 }
-.title {
+
+.menu-nav-button.MuiButton-root {
+    font-size: var(--tx-L);
+    font-weight: 100;
+    line-height: var(--pm-6XL);
```
Comment (sebaskleeen) [inline libs/atomic-elements/src/lib/components/NavTop/NavTop.scss:16]: line height is almost 100% related to font sizes, so probably some `var(--tx-<something>)` for this one probably `tx-L` like the font size above 👍

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx [hunk 2]
```diff
@@ -66,6 +67,40 @@ const defaultValue = {
...
 };
 
+const AppBar = styled(MuiAppBar)({
+  boxShadow: 'var(--shadow-elevation-high-key)',
+});
+
+const Button = styled(MuiButton)({
+  borderRadius: '0px',
+  "&:hover": {
+    background: '#03DAC6',
```
Comment (TreyJV) [inline libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:77]: Can we use the variable color? Should be `--secondary-color`

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx [hunk 3]
```diff
@@ -66,6 +67,40 @@ const defaultValue = {
...
+  },
+  "&:focus": {
+    background: '#03DAC6',
```
Comment (TreyJV) [inline libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:80]: Same here

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx [hunk 4]
```diff
@@ -66,6 +67,40 @@ const defaultValue = {
...
+    "&:hover": {
+      background: '#6200EE',
+    },
+  }
+});
+
+const MenuItem = styled(MuiMenuItem)({
+  "&:hover": {
+    background: '#018786',
+    color: '#FFFFFF',
```
Comment (TreyJV) [inline libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:92]: Can we use the variable color? Should be `--on-primary-color`

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx [hunk 5]
```diff
@@ -66,6 +67,40 @@ const defaultValue = {
...
+  }
+});
+
+const Paper = styled(MuiPaper)({
+  boxShadow: 'var(--shadow-elevation-high-key)',
+});
+
+const Toolbar = styled(MuiToolbar)({
+  background: '#3700B3',
```
Comment (TreyJV) [inline libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:101]: Can we use the variable color? Should be `--primary-color`

File: libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx (hunk 2 above)
Comment (mariokleen) [reply libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:77]: we don't know the color (secondary vs primary vs whatever), I thought we were going to use the invision colors for this, I can change it, but can you confirm?

Comment (mariokleen) [reply libs/atomic-elements/src/lib/components/NavTop/NavTop.tsx:77]: in some cases this makes the styles not "evident" e.g.

image
vs
image

I got the changes ready on my branch, I'll wait to hear back from you guys :) @EdisonKleeen @TreyJV @LauraKleeen

**Pull request merged**